spring-cloud-stream-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -81,147 +71,586 @@ public class StreamApplicationTests {
-`spring.cloud.stream.bindings.input=foo`, but that shorthand can only be used only when there are no other attributes
-to set on the binding. In other words,
-`spring.cloud.stream.bindings.input.destination=foo`,`spring.cloud.stream.bindings.input.partitioned=true` is a valid
-setup, whereas `spring.cloud.stream.bindings.input=foo`,`spring.cloud.stream.bindings.input.partitioned=true` is not.
+.Spring Cloud Stream Consumer Groups
+image::SCSt-groups.png[width=300,scaledwidth="50%"]
 
-===== Partitioning
+[[durability]]
+==== Durability
+The other concept that is related to the publish subscribe model of Spring Cloud Stream is durability. Again, part of the opinionated application model of Spring Cloud Stream, consumer group subscriptions are durable. This is to say that the binder implementation will ensure that group subscriptions are persistent and, once at least one subscription for a group has been created, that group will receive messages, even if they are sent while all the applications of the group were stopped. Anonymous subscriptions are non-durable by nature. For some binder implementations (e.g. Rabbit) it is possible to have non-durable group subscriptions.
+
+In general, it is preferable to always specify a consumer group when binding an application to a given destination. When scaling up a Spring Cloud Stream application, a consumer group must be specified for each of its input bindings, in order to prevent its instances from receiving duplicate messages (unless that behavior is desired, which is a less common use case).
 
-Spring Cloud Stream provides support for partitioning data between multiple instances of a given application. In a
-partitioned scenario, one or more producer apps will send data to one or more consumer apps, ensuring that data with
-common characteristics is processed by the same consumer instance. The physical communication medium (i.e. the broker
-topic or queue) is viewed as structured into multiple partitions. Regardless of whether the broker type is naturally
-partitioned (e.g. Kafka) or not (e.g. Rabbit), Spring Cloud Stream provides a common abstraction for implementing
-partitioned processing use cases in a uniform fashion.
+[[partitioning]]
+=== Partitioning
+
+Another native concept of Spring Cloud Stream is partitioning. Spring Cloud Stream provides support for partitioning data between multiple instances of a given application. In a partitioned scenario, one or more producer application instances will send data multiple consumer application instances, ensuring that data with common characteristics is processed by the same consumer instance. The physical communication medium (e.g. the broker topic) is viewed as structured into multiple partitions. This happens regardless of whether the broker type is naturally partitioned (e.g. Kafka) or not (e.g. Rabbit), Spring Cloud Stream provides a common abstraction for implementing partitioned processing use cases in a uniform fashion.
```

Comment:
> application instances will send data multiple consumer application instances

could be: "_application instances will send data to multiple consumer application instances_"